go/vt/vtgate/planbuilder/expr.go

```diff
@@ -90,6 +107,14 @@ func (pb *primitiveBuilder) findOrigin(expr sqlparser.Expr) (origin builder, pus
 			if isLocal && newOrigin.Order() > highestOrigin.Order() {
 				highestOrigin = newOrigin
 			}
+		case *sqlparser.ComparisonExpr:
+			if node.Operator == sqlparser.InStr || node.Operator == sqlparser.NotInStr {
```

Comment:
I don't understand most of this code, but my spidey sense is tingling here. Should this `if` not have a corresponding `else`?

Reply:
Never mind, I think I get it now.